src/ai/backend/gateway/kernel.py

```diff
@@ -275,7 +287,9 @@ def interrupt_wait(ctx: Any, event_name: str, agent_id: AgentId,
                 raise InvalidAPIParameters('Invalid group')
 
         api_version = request['api_version']
-        if (4, '20190315') <= api_version:
+        if (4, '20191119') <= api_version:
```

Comment:
Conventionally we keep the API version to the middle of triple months (one of 0315, 0615, 0915, or 1215).
Since we will make a breaking change for the naming of kernels and sessions in v5, let's make the version to `v5.20191215`.